package.json

```diff
@@ -29,26 +29,25 @@
     }
   },
   "dependencies": {
-    "@hapi/hoek": "^9.0.4",
-    "@hapi/topo": "^5.0.0",
+    "@hapi/hoek": "^9.0.4 || ^10.0.0 || ^11.0.0",
```

Comment:
`@hapi/hoek` at [v11.0.0](https://www.npmjs.com/package/@hapi/hoek/v/11.0.0) is deprecated, so it's better to start with `^11.0.1` in v11 range.